Dockerfile

```diff
@@ -17,7 +17,7 @@ COPY requirements.txt stale_repos.py /action/workspace/
 
 RUN python3 -m pip install --no-cache-dir -r requirements.txt \
     && apt-get -y update \
-    && apt-get -y install --no-install-recommends git-all=1:2.39.2-1.1 \
+    && apt-get -y install --no-install-recommends git \
```

Comment:
@zkoppert this was my fix to get builds to pass.  I imagine we'd want to pin `git` but I don't think we need `git-all`.  Do you know why we might need that? [github/contributors PRs](https://github.com/github/contributors/pull/175) experiencing same issue.  I imagine the others will also.

Reply:
Thanks for fixing this!

Looking at the [dependencies of git-all](https://packages.debian.org/sid/git-all) I don't see any need for anything other than the git package for what we are doing. A GUI or git-docs for example are not needed in the docker container.

Feel free to make those changes across all of our actions too.